frontend/packages/ceph-storage-plugin/src/components/modals/block-pool-modal/create-block-pool-modal.tsx

```diff
@@ -110,18 +114,12 @@ export const CreateBlockPoolModal = withHandlePromise((props: CreateBlockPoolMod
       <ModalTitle>{MODAL_TITLE}</ModalTitle>
       <ModalBody>
         <p>{MODAL_DESC}</p>
-        {state.poolStatus ? (
-          <div key="progress-modal">
-            <BlockPoolStatus status={state.poolStatus} name={state.poolName} error={errorMessage} />
-          </div>
-        ) : (
-          <BlockPoolBody
-            cephCluster={cephClusters[0]}
-            state={state}
-            dispatch={dispatch}
-            showPoolStatus
-          />
-        )}
```

Comment:
just removed pool status here and moved to body.tsx, now code is small and simple here